csv_provider/README.md

```diff
@@ -0,0 +1,43 @@
+# VSS Provider
+The aim of this script to provide and replay VSS data into a `kuksa.val` databroker. It is possible to define a sequence of VSS signals including timing delays in a CSV-file and then to apply this sequence towards a `kuksa.val` databroker with the provider.
```

Comment:
```suggestion
The aim of this script is to provide and replay VSS data into an instance of `kuksa.val` databroker. Therefore it takes a CSV file, containing pre defined sequences, consisting of VSS signals including timing delays, and provides it to the 'kuksa.val' databroker
```

Reply:
Thanks for the suggestion, I adapted the paragraph based on it.